pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -1456,6 +1460,27 @@ type PrivateCluster struct {
 	AdditionalEndpointServices []string `json:"additionalEndpointServices,omitempty"`
 }
 
+// InstanceSelector holds EC2 instance selector options
+type InstanceSelector struct {
+	// VCPUs specifies the number of vCPUs
+	VCPUs int `json:"vCPUs,omitempty"`
```

Comment:
the casing of `vCPUs` compared to `gpus` is a bit weird

Reply:
Yeah, I had highlighted this in the proposal: https://github.com/weaveworks/eksctl/blob/main/docs/proposal-007-instance-selector-dry-run.md#design

>Note: Because fields are case-sensitive, vCPUs may be difficult to get right, especially for users unfamiliar with Go naming conventions. However, vCPUs with the same case is standard enough that it's widely used by all major cloud providers. vcpus is easier to remember but it's inconsistent with the other field names.